packages/millicast-sdk/src/View.js

```diff
@@ -207,7 +273,8 @@ export default class View extends BaseWebRTC {
       subscriberData = await this.tokenGenerator()
       //  Set the iceServers from the subscribe data into the peerConfig
       this.options.peerConfig.iceServers = subscriberData?.iceServers
-      this.options.peerConfig.encodedInsertableStreams = supportsInsertableStreams
+      // We should not set the encodedInsertableStreams if the DRM and the frame metadata are not enabled
+      this.options.peerConfig.encodedInsertableStreams = supportsInsertableStreams && (this.isDRMOn || this.isFrameMetadataOn)
```

Comment:
`this.frameMetadataOn` isn't set anywhere or defined anywhere? It isn't a roadblock for right now but would need cleaning up before we release DRM with the SDK :)

Reply:
Yes, this flag is undefined now, and frame metadata is not compatible with DRM. But I am thinking we should have a flag to identify this feature.

Reply:
I completely agree that we should have a flag to identify this option :) Lets keep it, but absolutely, we can clean up and define later :)

Reply:
have updated with 'options.metadata'